meta-resin-common/classes/kernel-resin.bbclass

```diff
@@ -324,6 +325,10 @@ RESIN_CONFIGS[audit] = " \
     CONFIG_AUDITSYSCALL=n \
     "
 
+RESIN_CONFIGS_DEPS[governor] ?= " \
+    CONFIG_CPU_FREQ_DEFAULT_GOV_ONDEMAND=y \
+    "
```

Comment:
Does this work reliably? e.g. if we have a BSP that sets CONFIG_CPU_FREQ_DEFAULT_GOV_XXX also which gets used? Given these governors can all be built at once and enabled from userspace would it be better to make sure all the governors are built and allow the user to configure in their startup scripts?